website/docs/configure_hydra/workdir.md

```diff
@@ -36,16 +31,102 @@ Run output directory can contain user configuration variables:
 hydra:
   run:
     dir: outputs/${now:%Y-%m-%d_%H-%M-%S}/opt:${optimizer.type}
+```
+
+### Configuration for multirun
+
+Default multirun dir configurations:
+```yaml
+hydra:
+  sweep:
+    dir: multirun/${now:%Y-%m-%d}/${now:%H-%M-%S}
+    subdir: ${hydra.job.num}
+```
+Run the example application
+```bash
+python my_app.py --multirun a=a1,a2,a3 
+```
+
+would create working dir structure like:
+```bash
+$ tree -d multirun/
+multirun/
+└── 2021-03-10
+    └── 17-21-00
+        ├── 0
+        ├── 1
+        └── 2
+```
+Similar configuration patterns in run can be applied to config multirun dir as well.
+
+For example, multirun output directory grouped by job name, and sub dir by job num:
+```yaml
+hydra:
+  sweep:
+    dir: ${hydra.job.name}
+    subdir: ${hydra.job.num}
```

Comment:
Show the tree structure. this is a bit confusing (job.name, job.num).

Also, you can use column layout to put the config box next to the tree conf.
That would make things more compact.

(Do it only if you can make the two columns fit nicely without a scroll bar).

Reply:
Yea, good idea. Created columns to fix the configs :)